src/wasm-linker.cpp

```diff
@@ -210,6 +210,15 @@ void Linker::layout() {
     auto* func = out.wasm.getFunction(name);
     func->type = ensureFunctionType(getSig(func), &out.wasm)->name;
   }
+
+  // Export malloc and free whenever availble. JavsScript version of malloc has
```

Comment:
I don't think we need to do (1). We just need to handle the prefix difference (either before sending it, or when receiving it).

I don't think the size of `EXPORTED_FUNCTIONS` would be a problem in practice. So I don't think we need to add an s2wasm option to read from a file, or we could leave that for later.

But if doing the more proper fix would slow you down and you'd prefer to just land this (with a TODO comment) and fix things later, I'm ok with that too, I don't want to hold things up.